include/tss2/tss2_rc.h

```diff
@@ -0,0 +1,14 @@
+#ifndef TSS2_RC_H
```

Comment:
License tag and copyright tag are missing

Reply:
Now it's just the copyright. They're pointless, and I would make it a habit to not accept them in new files.